worker/updateChorals.js

```diff
@@ -194,6 +186,6 @@ function sendChoralData(choral, computed_data){
     if(body.error){
       console.log(body.error)
```

Comment:
I think you are right but I think that's another issue. We should add in logging in a bunch of places.